src/docs/get-started/install/_get-sdk.md

```diff
@@ -111,6 +81,30 @@ process.
 Once you have installed any missing dependencies, run the `flutter doctor`
 command again to verify that you’ve set everything up correctly.
 
+### Downloading straight from GitHub instead of using a tarball
+
+_This is only suggested for advanced use cases._
+
+You can also use git directly instead of downloading the prepared tarball. For example,
+to download the stable branch:
+    
+```terminal
+$ git clone https://github.com/flutter/flutter.git -b stable
+```
+
+If you do this, you may also wish to pre-download development binaries that are included
```

Comment:
Should we insert another `flutter doctor` before `flutter precache` here? And/or general instructions about ensuring all system requirements are present before continuing.

Reply:
done